concepticondata/conceptlists/Lynott-2009-423.tsv

```diff
@@ -0,0 +1,424 @@
+Lynott-2009-423-32	32	blotchy			1	4.61904761904762	1.66666666666667	0.428571428571429	0.238095238095238	0.380952380952381	0.804747816162957	1.79814719456816	0.925820099772551	0.889087279447969	1.07126982951031	Visual	0.597402597402597	0	0	7
+Lynott-2009-423-33	33	blue	BLUE	837	1	4.76190476190476	0.238095238095238	0.761904761904762	0.19047619047619	0	0.538958431120796	0.624880941040924	1.22085060121056	0.511766315719159	0	Visual	0.8	104	4.65396035015752	4
+Lynott-2009-423-34	34	blunt	BLUNT	379	1	4.57142857142857	4.71428571428571	0.904761904761905	0.19047619047619	0.25	0.507092552837111	0.643650304346789	1.26114082896249	0.872871560943969	0.550119604220181	Haptic	0.425531914893617	5	1.79175946922806	5
+Lynott-2009-423-35	35	boiling	BOIL (OF LIQUID)	792	1	4.42857142857143	4.04761904761905	2.76190476190476	1	2.0952380952381	0.74642002729218	1.4309504001254	2.09534631755139	1.54919333848297	2.16575072214606	Visual	0.239202657807309	9	2.30258509299405	7
+Lynott-2009-423-36	36	booming			1	2.0952380952381	0.952380952380952	4.57142857142857	0.095238095238095	0.095238095238095	1.67047184665776	1.20317041503648	0.810643483377778	0.300792603759119	0.436435780471985	Auditory	0.573170731707317	3	1.38629436111989	7
+Lynott-2009-423-37	37	bouncy			1	3.57142857142857	3.66666666666667	2	0	0	1.20712172424444	1.55991452757301	1.61245154965971	0	0	Haptic	0.396907216494845	1	0.693147180559945	6
+Lynott-2009-423-38	38	brackish	BRACKISH	70	0.095238095238095	2	3	1.5	2	1.5	1.4142135623731	1.4142135623731	2.12132034355964	2.82842712474619	2.12132034355964	Haptic	0.15	1	0.693147180559945	8
+Lynott-2009-423-39	39	branching			0.952380952380952	4.1	2.45	1.2	0.1	0.05	0.967906041546987	1.57195821559574	1.67332005306815	0.307793505625546	0.223606797749979	Visual	0.512658227848101	3	1.38629436111989	9
+Lynott-2009-423-40	40	breakable			1	4.28571428571429	4.23809523809524	2.0952380952381	0.095238095238095	0.095238095238095	0.902377811277357	0.889087279447968	1.75797502555531	0.300792603759119	0.300792603759119	Visual	0.387665198237885	0	0	9
+Lynott-2009-423-41	41	breezy			1	3	2.76190476190476	2.95238095238095	0.571428571428571	0.047619047619048	1.34164078649987	1.78619041271534	1.35927151357595	1.32557265253281	0.218217890235992	Visual	0.316326530612245	1	0.693147180559945	6
+Lynott-2009-423-42	42	bright	BRIGHT	679	1	5	0.19047619047619	0.142857142857143	0.095238095238095	0.047619047619048	0	0.679635756787974	0.478091443733757	0.436435780471985	0.218217890235992	Visual	0.904347826086957	54	4.00733318523247	6
+Lynott-2009-423-43	43	brilliant	GLARING	2228	1	3.95238095238095	2.19047619047619	3	2.0952380952381	2.52380952380952	1.28359613882991	2.27198256197198	1.87082869338697	2.0713464679952	2.11232212550661	Visual	0.134948096885813	35	3.58351893845611	9
```

Comment:
I think this could justify a new concept? A 'brilliant diamond' is not necessarily 'glaring'?

Reply:
The one thing currently mapped to GLARING is "brilliant, glaring". But _brilliant_ is a difficult and ambiguous word anyway.

Reply:
Can we use the definition by the Collins dictionary?
BRILLIANT = "(Of a color) having a high saturation and reflecting a considerable amount of light"